lib/types/Template.d.ts

```diff
@@ -8,6 +8,8 @@ declare interface Template extends BaseTemplate {
 	/** Step by step */
 	listInComponentTemplates: boolean;
 	listInCustomTemplates: boolean;
+	/** Extra packages to install when adding to project */
+	packages: string[];
```

Comment:
We should update Extensibility Wiki (the table).
Also we need to update generate template in all project libraries with the packages.

Reply:
Updated https://github.com/IgniteUI/igniteui-cli/wiki/Extensibility-Support#configure-templatejson